README.md

```diff
@@ -36,7 +36,7 @@ As audio/video packets are streamed from a source to a destination device, SRT d
 * [Why SRT Was Created](docs/why-srt-was-created.md)
 * [SRT Protocol Technical Overview](https://github.com/Haivision/srt/files/2489142/SRT_Protocol_TechnicalOverview_DRAFT_2018-10-17.pdf)
 * SRT Cookbook: [website](https://srtlab.github.io/srt-cookbook), [GitHub](https://github.com/SRTLab/srt-cookbook)
-* SRT RFC: [txt](https://haivision.github.io/srt-rfc/draft-sharabayko-mops-srt.txt), [html](https://haivision.github.io/srt-rfc/draft-sharabayko-mops-srt.html), [GitHub](https://github.com/Haivision/srt-rfc)
+* SRT RFC: [txt](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00), [html](https://haivision.github.io/srt-rfc/draft-sharabayko-mops-srt.html), [GitHub](https://github.com/Haivision/srt-rfc)
```

Comment:
Maybe it is worth marking the version (IETF or working) in use.
Consider also https://github.com/Haivision/srt-rfc/pull/96 which renames the file in SRT RFC GitHub.

```suggestion
* SRT RFC: [Latest IETF Draft](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00), [Latest working copy](https://haivision.github.io/srt-rfc/draft-sharabayko-srt.html), [GitHub](https://github.com/Haivision/srt-rfc)
```